pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -70,7 +70,11 @@ def get_highest_version(self, obj):
                 "version": collection.version,
             },
         )
-        return {"href": href, "version": str(collection.version)}
+        return {
+            "href": href,
+            "version": str(collection.version),
+            "requires_ansible": collection.requires_ansible,
```

Comment:
@bmbouter I added `requires_ansible` on the NamedTuple to get it for the highest version and display it at collection level:
![image](https://user-images.githubusercontent.com/17153022/106493086-74f53b80-6497-11eb-8eeb-4cdbe252777f.png)

Reply:
I only expected the `requires_ansible` to be on the serializer for `collections/<str:namespace>/<str:name>/versions/<str:version>/`. Maybe I didn't understand the requirements though. What motivated delivering this data there?